process/collaboration_tools.md

```diff
@@ -0,0 +1,188 @@
+# Academy Software Foundation community collaboration tools
+
+Academy Software Foundation maintains a number of tools to enable collaboration amongst community participants. To help navigate these tools, this document provides a brief outline of how to use them and best practices.
+
+## Slack
```

Comment:
A couple of additional notes I would suggest:

- point out that some projects (OCIO notably) have their own Slack instance
- a reminder that Slack messages will expire, so by nature the medium is ephemeral: useful long term content should be transferred to a project repo, its documentation, or the Confluence wiki